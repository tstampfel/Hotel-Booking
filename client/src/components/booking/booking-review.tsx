// @flow
import { motion } from "framer-motion";
import * as React from "react";
type Props = {};
export function BookingReview(props: Props) {
  return (
    <motion.div
      key="booking-review"
      initial={{ opacity: 0 }}
      animate={{ opacity: 1 }}
      exit={{ opacity: 0 }}
      className="booking-review"
    >
      Review
    </motion.div>
  );
}
