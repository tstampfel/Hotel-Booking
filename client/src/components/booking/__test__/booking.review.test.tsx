import { render } from "@testing-library/react";
import React from "react";
import { BookingReview } from "../booking-review";

describe("Booking components test", () => {
  test("<BookingReview /> matches snapshot", () => {
    const component = render(<BookingReview />);
    expect(component.container).toMatchSnapshot();
  });
});
