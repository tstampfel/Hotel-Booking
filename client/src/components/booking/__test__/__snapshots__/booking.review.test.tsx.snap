// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking components test <BookingReview /> matches snapshot 1`] = `
<div>
  <div
    class="booking-review"
    style="opacity: 0;"
  >
    Review
  </div>
</div>
`;
